WordPress/WordPressTest/WPAccount+LookupTests.swift

```diff
@@ -47,40 +47,40 @@ class WPAccountLookupTests: CoreDataTestCase {
     }
 
     func testLookupAccountByUUIDReturnsNilForInvalidAccount() throws {
-        AccountBuilder(contextManager).build()
+        makeAccount()
         try XCTAssertNil(WPAccount.lookup(withUUIDString: "", in: contextManager.mainContext))
     }
 
     func testLookupAccountByUUIDReturnsAccount() throws {
         let uuid = UUID().uuidString
-        AccountBuilder(contextManager).with(uuid: uuid).build()
+        makeAccount { $0.with(uuid: uuid) }
```

Comment:
As you can see below, `makeAccount` can be called either with no argument or passed a closure with an `AccountBuilder` input and `AccountBuilder` output: `(AccountBuilder) -> AccountBuilder`.

The `{ $0.with(uuid: uuid) }` syntax leverages is a condensed version of:

```swift
{ builder in 
  return builder.with(uuid: uuid)
}
```

where `$0` allows us skip naming the input parameter:

```swift
{ return $0.with(uuid: uuid) }
```

and the `return` can be omitted because the compiler infers it from the closure declaring that it has a return type and the only code in it matching that return type

```swift
{ $0.with(uuid: uuid) }
```

I think it's neat and [sugary](https://en.wikipedia.org/wiki/Syntactic_sugar). But let me know if it's too arcane.